lib/slack-ruby-bot-server/ext.rb

```diff
@@ -0,0 +1,3 @@
+%w(bson/object_id grape/sort_extension slack-ruby-bot).each do |ext|
```

Comment:
True. This made a little more sense when I was writing the code and the list had 2 more entries in it.